### ISSUE TITLE
Sniff raw NRF24 frames

### ISSUE THREAD

**Issue by drakylar** [mame82/LOGITacker#42]
Is it available to sniff raw nrf24 frames for unknown devices without changing code of toolkit? Or can you add it to TODO list?

**Comment by myfreeweb** [comment]
Yes, comment/remove `#define LOGITECH_FILTER` in `esb_illegalmod/nrf_esb_illegalmod.c`
